tests/structured_conf/data/attr_classes.py

```diff
@@ -393,15 +393,15 @@ class Str2User(Dict[str, User]):
     class Str2StrWithField(Dict[str, str]):
         foo: str = "bar"
 
-    @attr.s(auto_attribs=True)
-    class Str2IntWithStrField(Dict[str, int]):
-        foo: str = "bar"
```

Comment:
The idea with foo here is that you can have a fields in dataclass extending Dict.

make that field an int to conform with the superclass key type instead of removing it.